### PR TITLE
Add an entry for DevOps with OpenShift

### PR DESCRIPTION
**Pull request by jankleinert**

### CODE REVIEW

File: docs/devops-with-openshift.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# DevOps with OpenShift
```
Comment (thoraxe) [inline docs/devops-with-openshift.md]: Minor nit - we have not been using `#` / `<h1>` tags because the title is already in thirty places.

**Review by thoraxe — CHANGES_REQUESTED** [review]
One teeny change requested

File: docs/devops-with-openshift.md (hunk 1 above)
Comment (jankleinert) [reply docs/devops-with-openshift.md]: @thoraxe updated!

**Comment by thoraxe** [comment]
Needs rebase

**Comment by jankleinert** [comment]
I'm going to enter a fresh PR

**Pull request closed without merging**